src/main/resources/input

```diff
@@ -0,0 +1,9 @@
+type,fruit,quantity
+b,banana,20
+b,apple,100
+s,banana,100
+p,banana,13
+r,apple,10
+p,apple,20
+p,banana,5
+s,banana,50
```

Comment:
As I understand, the report should be in report file and it should be *.csv

Also, you should take info from DB file, like database.csv